salesforce_api/gsheet-new-row-to-sfdc-new-contact/Readme.md

```diff
@@ -0,0 +1,79 @@
+Example to create new contacts in the Salesforce pricebook using google sheets and Salesforce integration.
```

Comment:
Shouldn't we say "Google Sheets" instead of "google sheets"? Applicable in multiple places.

Reply:
Updated in [900e154](https://github.com/ballerina-guides/integration-samples/pull/31/commits/900e154cf4d53db6e093f4fd6d5b450900bbafa3)